code/modules/mob/living/simple_animal/friendly/mouse.dm

```diff
@@ -113,5 +113,16 @@
 	foodtype = GROSS | MEAT | RAW
 	grind_results = list("blood" = 20, "liquidgibs" = 5)
 
+/obj/item/reagent_containers/food/snacks/deadmouse/attackby(obj/item/I, mob/user, params)
+	if(I.is_sharp() && user.a_intent == INTENT_HARM)
+		if(isturf(loc))
+			new /obj/item/reagent_containers/food/snacks/meat/slab/mouse(loc)
```

Comment:
You should be making use of the butcher component

Reply:
The butcher proc only exists for living mobs. This workaround allows for the duality of having a mouse as a food item, and being able to slice the mouse into a fillet.

Reply:
Then make it work for this as well if you need to